### PR TITLE
force focus for some fields

### PR DESCRIPTION
**Pull request by smk762**
partial fix for https://github.com/KomodoPlatform/atomicDEX-Desktop/issues/1494

auto focuses on input fields:
- login screen password input
- asset search fields on portfolio and wallet sidebar
- on recover seed password input

### CODE REVIEW

File: atomic_defi_design/Dex/Components/DexAppPasswordField.qml [hunk 1]
```diff
@@ -31,6 +32,9 @@ DexAppTextField
     field.leftPadding: 75
     field.rightPadding: 60
     field.placeholderText: qsTr("Type password")
+    Component.onCompleted: {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/DexAppPasswordField.qml:35]: Coding style

File: atomic_defi_design/Dex/Components/SearchField.qml [hunk 2]
```diff
@@ -51,5 +50,8 @@ Rectangle
         placeholderText: qsTr("Search")
         placeholderTextColor: Dex.CurrentTheme.textPlaceholderColor
         font.pixelSize: 14
+        Component.onCompleted: {
```
Comment (SylEze) [inline atomic_defi_design/Dex/Components/SearchField.qml:53]: Coding style

**Review by SylEze — CHANGES_REQUESTED** [review]

**Review by SylEze — APPROVED** [review]

**Comment by tonymorony** [comment]
@smk762 there is conflict now `atomic_defi_design/Dex/Portfolio/Portfolio.qml`

**Review by tonymorony — CHANGES_REQUESTED** [review]
^^

**Comment by smk762** [comment]
> @smk762 there is conflict now `atomic_defi_design/Dex/Portfolio/Portfolio.qml`

thanks, sorted.

**Review by tonymorony — APPROVED** [review]
saving some clicks, thanks!

**Pull request merged**